partiql/tests/snapshots/pretty__graph_etc_6.snap

```diff
@@ -0,0 +1,32 @@
+---
+source: partiql/tests/pretty.rs
+expression: doc
+---
+========================================================================================================================================================================================================
+SELECT * FROM (g MATCH ( (x)-[e]->*(y) ) |+| (z) ~ (q) )
```

Comment:
perhaps can call out somewhere that we choose to make the `GRAPH_TABLE` keyword optional? SQL 2023's BNF definition makes it required

```
<table primary> ::=
!! All alternatives from ISO/IEC 9075-2
| <graph table derived table>
<graph table derived table> ::=
<graph table> [ <correlation or recognition> ]
<graph table> ::=
GRAPH_TABLE
<left paren> <graph reference> MATCH <graph pattern> <graph table shape>
<right paren>
```

Reply:
https://github.com/partiql/partiql-lang/issues/99

Reply:
This is a result of treating `MATCH` 'just' like any other expression. E.g., `PartiQL` allows `SELECT * FROM <anything that is a PartiQL expression here>` and `(<expr> MATCH <graph-pattern>)` is defined as an expression.